apps/condo/domains/billing/constants/errors.js

```diff
@@ -1,7 +1,37 @@
-const { WRONG_FORMAT } = require('@condo/domains/common/constants/errors')
+const { GQLErrorCode: { BAD_USER_INPUT } } = require('@open-condo/keystone/errors')
+
+const { WRONG_FORMAT, NOT_FOUND } = require('@condo/domains/common/constants/errors')
 
 const BILLING_INTEGRATION_WRONG_GROUP_FORMAT_ERROR = `[${WRONG_FORMAT}:BillingIntegration:group] group should be a sequence of lowercase latin characters`
 
+const SEND_RESIDENT_MESSAGE_ERRORS = {
```

Comment:
We define our errors inside the Service file. You need to move it or change our global style (but no in this PR)

Reply:
I moved them because I reuse them in tests and was unable to import them from the service

Reply:
What about others cases? We have tests for errors